### ISSUE TITLE
[member-ordering] conflict with TSLint ("tslint:latest" config)

### ISSUE THREAD

**Issue by iliyaZelenko** [bradzacher/eslint-plugin-typescript#205]
My `tslint.json`:

```JSON
{
  "extends": ["tslint:latest"]
}
```

My `.eslintrc.js` config does not contain rule modifications for `member-ordering` rule. See [screenshot](https://i.imgur.com/P8xyY6e.png).

My code:

```TS
class Class {
  public secondPropery = 'test'
  // ESLint: Member propery should be declared before all public field definitions. (typescript/member-ordering)
  private propery = 'test'

  public constructor () {
    // so that there are no errors about not using properties
    console.log(this.propery, this.secondPropery)
  }
}
```
If I put `private propery` before `public secondPropery`, there will be an error from TSLint:

> TSLint: Declaration of public instance field not allowed after declaration of private instance field. Instead, this should come at the beginning of the class/interface. (member-ordering)

**Expected Result**

Behavior like TSLint. 

Judging by [this post](https://stackoverflow.com/a/310967/5286034) TSLint uses the correct behavior.

**Actual Result**

Errors.

**Versions**

| package                    | version |
| -------------------------- | ------- |
| `eslint-plugin-typescript` | `0.14.0` |
| `typescript-eslint-parser` | `21.0.2` |
| `typescript`               | `3.2.1` |

![image](https://user-images.githubusercontent.com/13103045/49524138-d9b2ac80-f8b3-11e8-801f-2c9bee6db7fa.png)

**Comment by bradzacher** [comment]
Could you please post your config for the rule?
I ran this against 0.14.0 with default config and it passed fine.

```
class Class {
    public secondPropery = 'test'
    private propery = 'test'

    public constructor () {}
}
```

**Comment by bradzacher** [comment]
Looking at your config - you have (trimmed to useful info..)

```
{
    plugins: ['typescript'],
	extends: ['typescript'],
	rules: [
		// no config for the rule		
    ],
}
```

It looks like you're using `eslint-config-typescript`, [which has specifically configured the order](https://github.com/weirdpattern/eslint-config-typescript/blob/master/src/typescript.js#L54-L68) to be private then public.

If you want the ordering to match tslint - either configure tslint with that order, or overwrite the config with the tslint order.
